db/Seed/seedbuilder.js

```diff
@@ -4,21 +4,28 @@ if (process.env.NODE_ENV !== 'production')
 
 const Promise = require('bluebird');
 // db + models
+
 const db = require('../../db/');
 const therapistModel  = db.model('therapist');
 const patientModel  = db.model('patient');
 const exerciseModel = db.model('exercise');
-const planModel = db.model('plan')
-const treatmentModel = db.model('treatment')
-const workoutModel = db.model('workout')
-
+const planModel = db.model('plan');
+const treatmentModel = db.model('treatment');
+const workoutModel = db.model('workout');
 // seeds
 const TherapistList = require('./therapistSeed');
 const PatientList = require('./patientSeed');
 const ExerciseList = require('./exerciseSeed');
-const PlanList = require('./planSeed')
-const TreatmentList = require('./treatmentSeed')
-const WorkoutList = require('./workoutSeed')
+const PlanList = require('./planSeed');
+const TreatmentList = require('./treatmentSeed');
+const WorkoutList = require('./workoutSeed');
+
+const data = {
```

Comment:
Im not sure what it suppose to look like? It works when I build my seed? I will pull from master again and see whats going on...